run_fastsurfer.sh

```diff
@@ -220,6 +227,9 @@ SURFACE PIPELINE:
                             to exist already in this case.
   --3T                    Use the 3T atlas for talairach registration (gives better
                             etiv estimates for 3T MR images, default: 1.5T atlas).
+  --edits                 Disable the check for existing recon-surf.sh run and enables
+                            edits of mri/wm.mgz and brain.finalsurfs.mgz by manedit
+                            files as well as FreeSurfer-style WM control points.
```

Comment:
I think it is strange to have the flag --edits in the help text at multiple places. I think it should be a single help block explaining all of its effects in one location.